docs/microbit_micropython_api.rst

```diff
@@ -70,10 +73,39 @@ The LED display is exposed via the `display` object::
     # written messages).
     display.scroll(string, delay=400)
 
+Microphone **V2**
+----------
+
+The Microphone is accessed via the `microphone` object::
+
+    # Value to represent the transition of sound events, from `quiet` to `loud`
+    # like clapping or shouting.
+    SoundEvent.LOUD = SoundEvent('loud')
+    # Value to represent the transition of sound events, from `loud` to `quiet`
+    # like speaking or background music.
+    SoundEvent.QUIET = SoundEvent('quiet')
```

Comment:
Because the methods below this are all inside microbit.microphone, this could be interpreted as being `microbit.microphone.SoundEvent.LOUD`, when it's directly in the microbit module `microbit.SoundEvent.LOUD`.
We should probably have a different section on top of microphone for SoundEvent.

Reply:
I have done this in https://github.com/bbcmicrobit/micropython/pull/692/commits/8f52c9728fe49c9219b1e54c7258b5bb4701d6c1 Do you think this list is okay as it is. I wonder if it should either be alphabetical by heading or that Images are grouped with the LED display and Sounds with the pins?

Reply:
I think this is good, SoundEvent in this case should go right next to microphone, or it's too hard to follow.

The only thing is that is not rendered correctly:

![image](https://user-images.githubusercontent.com/29712657/100787020-ea4a7e80-340a-11eb-9c6c-5ae408fde037.png)

Reply:
Thanks. Fixed the formating